tests/e2e_v2/upgrade/raycluster_sdk_upgrade_sleep_test.py

```diff
@@ -0,0 +1,177 @@
+import requests
+from time import sleep
+
+from codeflare_sdk import (
+    Cluster,
+    ClusterConfiguration,
+    TokenAuthentication,
+    get_cluster,
+)
+from codeflare_sdk.ray.client import RayJobClient
+
+from tests.e2e.support import *
+
+
+from codeflare_sdk.common import _kube_api_error_handling
+
+namespace = "test-ns-rayupgrade-sleep"
+# Global variables for kueue resources
+cluster_queue = "cluster-queue-mnist"
+flavor = "default-flavor-mnist"
+local_queue = "local-queue-mnist"
+
+
+# Creates a Ray cluster , submit RayJob mnist script long running
+class TestSetupSleepRayJob:
```

Comment:
What is the difference between this and the previous test of mnist job submission?